components/Order/models.rst

```diff
@@ -0,0 +1,178 @@
+
+Adjustments have the following properties:
+
++-------------------+-----------------------------------------+-----------------------------------+
+| Attribute         | Description                             | Returned value                    |
++===================+=========================================+===================================+
+| id                | Unique id of the adjustment             | mixed                             |
++-------------------+-----------------------------------------+-----------------------------------+
+| order             | Reference to Order                      | OrderInterface                    |
++-------------------+-----------------------------------------+-----------------------------------+
+| orderItem         | Reference to OrderItem                  | OrderItemInterface                |
++-------------------+-----------------------------------------+-----------------------------------+
+| label             | Type of the adjustment (e.g. "tax")     | string                            |
++-------------------+-----------------------------------------+-----------------------------------+
+| description       | e.g. "Clothing Tax 9%"                  | string                            |
++-------------------+-----------------------------------------+-----------------------------------+
+| amount            | Adjustment amount                       | int                               |
++-------------------+-----------------------------------------+-----------------------------------+
+| neutral           | Boolean flag of neutrality              | bool                              |
++-------------------+-----------------------------------------+-----------------------------------+
+| locked            | Adjustment lock (prevent from deletion) | bool                              |
++-------------------+-----------------------------------------+-----------------------------------+
+| originId          | Origin id of the adjustment             | int                               |
++-------------------+-----------------------------------------+-----------------------------------+
+| originType        | Origin type of the adjustment           | string                            |
++-------------------+-----------------------------------------+-----------------------------------+
+| createdAt         | Date when adjustment was created        | \\DateTime                        |
++-------------------+-----------------------------------------+-----------------------------------+
+| updatedAt         | Date of last change                     | \\DateTime                        |
++-------------------+-----------------------------------------+-----------------------------------+
+
+.. note::
+
+    This model implements the :ref:`component_order_model_adjustment-interface`
+
+.. _component_order_model_comment:
+Comment
+-------
+
+**Comment** object represents an comment to the order.
```

Comment:
`a comment`, assigned to the order or about the order, something along those lines.